platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,41 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
+ Creates a set of options with the minimum required information.
+ 
  @param styleURL the style url to use
  @param camera the camera settings
  @param size the image size
  */
 - (instancetype)initWithStyleURL:(NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
 
-#pragma mark - Configuring the map
+#pragma mark - Configuring the Map
 
 /**
   The style URL for these options.
  */
-@property (nonatomic, readonly) NSURL* styleURL;
+@property (nonatomic, readonly) NSURL *styleURL;
 
 /**
- The zoom. Default is 0.
+ The zoom level. Default is 0.
  */
-@property (nonatomic) double zoom;
+@property (nonatomic) double zoomLevel;
 
 /**
  The `MGLMapcamera` options to use.
  */
-@property (nonatomic) MGLMapCamera* camera;
+@property (nonatomic) MGLMapCamera *camera;
 
 /**
- A region to capture. Overrides the center coordinate
- in the mapCamera options if set
+ The cooordinate rectangle that encompasses the bounds to capture. Overrides the center coordinate
+ in the mapCamera options if set.
  */
-@property (nonatomic) MGLCoordinateBounds region;
+@property (nonatomic) MGLCoordinateBounds coordinateBounds;
 
-#pragma mark - Configuring the image
+#pragma mark - Configuring the Image
 
 /**
- The size of the output image. Minimum is 64x64
+ The size of the output image. Minimum is 64x64 pixels.
```

Comment:
Pixels as in resolution-dependent pixels? Or points, by which most images are measured on Apple platforms? I assume the latter:

> The size of the output image.
> 
> The image may be no less than 64 points wide and no less than 64 points tall.

But is “and” correct? Or is it possible to create an image narrower than 64 points but taller than 64 points?

Reply:
I talked to @frederoni and he clarified that is pixels according to a restriction in OpenGL ES1-2-3 IIRC. Since is the minimum is squared shaped.

Reply:
So the largest image you can create using this API is 32×32 points, about the size of a tab bar button?